templates/docs/patterns/tooltips.md

```diff
@@ -36,6 +36,31 @@ In some cases you may need the tooltip element to exist outside of the element i
 View example of the detached tooltips pattern
 </a></div>
 
+## Accessibility
+
+{# copy doc: https://docs.google.com/document/d/1Pgk4JYHIIbdkFmAzrPBKO1uNQnzmAeXqwTKtoiajoCA/edit# #}
+
+### How it works
+
+A tooltip is text that appears in a small overlay on demand, usually when hovering over the thing it describes. It is hidden by default, and becomes available on hover or focus of the control it describes. It should provide information that isn’t self explanatory or well known.
+
+### Considerations
+
+This component strives to follow [WCAG 2.1 (level AA) guidelines](https://www.w3.org/TR/WCAG21/), and care must be taken to ensure this effort is maintained when the component is implemented across other projects. This section offers advice to that effect:
+
+- Be sure to include the `aria-describedby` attribute with the `id` of the description.
+- They should be discoverable, not placed over plain text.
```

Comment:
Not sure, what "not placed over plain text" means here?

Reply:
I think it means don't use something which isn't obviously a tooltip, like a word in the middle of a paragraph. I'll reword this to make it more clear